test/unit/model/v2/test_platforms_section.py

```diff
@@ -149,6 +153,8 @@ def _model_platforms_docker_errors_section_data():
                 int(),
             ],
             'env': str(),
+            'restart_policy': str(),
+            'restart_retries': int(),
```

Comment:
These should be invalid options so the test fails and we capture it failed as you did below.

Reply:
Done.